tests/4-test-leafref/src/app/Client.java

```diff
@@ -97,40 +91,40 @@ public static void main(String[] args) throws IOException, JNCException {
         Leafref.enable();
         client.init();
         NodeSet configs = client.getConfig();
-        
+
         // Get (first) config with name "c"
         C cConfig = getCConfig(configs);
-        
+
         // Clone a backup configuration for rollback purposes
         YangElement backup = cConfig.clone();
 
         String configAsXML = cConfig.toXMLString();
         System.out.println("Initial config:\n" + configAsXML);
         System.out.println("HashCode of the config XML string: " +
                 configAsXML.hashCode());
-        
+
         ElementChildrenIterator lIterator = cConfig.l1Iterator();
         while (lIterator.hasNext()) {
             L1 h = (L1) lIterator.next();
-            
+
             // Internal representations of key1 and key2
             YangString key1Value = h.getKey1Value();
-            YangString key2Value = h.getKey2Value();
-            
+            // YangString key2Value = h.getKey2Value();
```

Comment:
remove

Reply:
rolled back the change - left for later cleanup/code enhancement...